.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+version: '3'
+services:
+  web:
+    # Uncomment the next line to use a non-root user for all processes. You can also
+    # simply use the "remoteUser" property in devcontainer.json if you just want VS Code
+    # and its sub-processes (terminals, tasks, debugging) to execute as the user. On Linux,
+    # you may need to update USER_UID and USER_GID in .devcontainer/Dockerfile to match your
+    # user if not 1000. See https://aka.ms/vscode-remote/containers/non-root for details.
+    # user: node
+
+    build: 
+      context: .
+      dockerfile: Dockerfile
+      args:
+        VARIANT: 12
+  
+    volumes:
+      - ..:/workspace:cached
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+    
+    links:
+      - mongo
+
+  mongo:
+    image: mongo
+    restart: unless-stopped
+    volumes:
+      - /data/db
```

Comment:
I know this was generated but shouldn't there be some mapping to a local folder where our dev data would be persisted? So when I during development create some data in my local Mongo container, I'd like to have them persisted so I can use those data also a few days later when my local Mongo container is restarted or rebuilt.

Reply:
you can create names volume and create folder in project

Reply:
I updated the comment, please check it out again 😁